tests/acceptance/characters/control/test_rpc_control_federated.py

```diff
@@ -0,0 +1,82 @@
+"""
+ This file is part of nucypher.
+
+ nucypher is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ nucypher is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+from base64 import b64encode
+
+import pytest
+
+from nucypher.characters.control.interfaces import BobInterface, EnricoInterface
+from nucypher.crypto.powers import DecryptingPower, SigningPower
+from nucypher.policy.collections import DecentralizedTreasureMap
+from tests.acceptance.characters.control.test_rpc_control_blockchain import validate_json_rpc_response_data
+
+
+def test_bob_rpc_character_control_join_policy(bob_rpc_controller, join_control_request, enacted_blockchain_policy):
```

Comment:
There's already a `tests/integration/characters/control/test_rpc_control_federated.py`. Is it possible this is something left out from a rebase?